src/NUnitTestAdapter/NUnit3TestAdapter.csproj

```diff
@@ -9,7 +9,7 @@
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>NUnit.VisualStudio.TestAdapter</RootNamespace>
-    <AssemblyName>NUnit.VisualStudio.TestAdapter</AssemblyName>
+    <AssemblyName>NUnit3.VisualStudio.TestAdapter</AssemblyName>
```

Comment:
I dislike an assemly name that looks like a namespace but isn't one. I think I'll just go with NUnit3TestAdapter to match the project name.